integration_test/retryableData.test.ts

```diff
@@ -167,31 +169,29 @@ describe('RevertData', () => {
       if (!parsed) throw err
 
       expect(parsed.callValueRefundAddress, 'callValueRefundAddress').to.eq(
-        depositParams.retryableData.callValueRefundAddress
+        await l1Signer.getAddress()
       )
-      expect(parsed.data, 'data').to.eq(depositParams.retryableData.callData)
+      // expect(parsed.data, 'data').to.eq(depositParams.retryableData.callData)
       expect(parsed.deposit.toString(), 'deposit').to.eq(
-        depositParams.l2GasCostsMaxTotal.toString()
+        depositParams.core.value
       )
       expect(parsed.excessFeeRefundAddress, 'excessFeeRefundAddress').to.eq(
-        depositParams.retryableData.excessFeeRefundAddress
+        await l1Signer.getAddress()
       )
-      expect(parsed.from, 'from').to.eq(depositParams.retryableData.sender)
+      expect(parsed.from, 'from').to.eq(await l1Signer.getAddress())
       expect(parsed.gasLimit.toString(), 'gasLimit').to.eq(
-        depositParams.l2GasLimit.toString()
+        RetryableDataTools.ErrorTriggeringParams.gasLimit
       )
 
-      expect(parsed.l2CallValue.toString(), 'l2CallValue').to.eq(
-        depositParams.retryableData.l2CallValue.toString()
-      )
+      expect(parsed.l2CallValue.toString(), 'l2CallValue').to.eq(constants.Zero)
 
       expect(parsed.maxFeePerGas.toString(), 'maxFeePerGas').to.eq(
-        depositParams.l2MaxFeePerGas.toString()
-      )
-      expect(parsed.maxSubmissionCost.toString(), 'maxSubmissionCost').to.eq(
-        depositParams.l2SubmissionFee.toString()
+        RetryableDataTools.ErrorTriggeringParams.maxFeePerGas
       )
-      expect(parsed.to).to.eq(depositParams.retryableData.destination)
+      // expect(parsed.maxSubmissionCost.toString(), 'maxSubmissionCost').to.eq(
```

Comment:
we could have a function to parse the tx request and get these values (then test them here)